src/index.js

```diff
@@ -448,14 +454,24 @@ export default class Carousel extends Component {
     if (transition !== 'fade') {
       const leftPos = leftOffset + dragOffset;
       trackStyle = { ...trackStyle,
-        transform: `translateX(${isRTL ? -leftPos : leftPos}px)`,
+        ...isVertical && { transform: `translateY(${isRTL ? -leftPos : leftPos}px)` },
+        ...!isVertical && { transform: `translateX(${isRTL ? -leftPos : leftPos}px)` },
         transition: transitionDuration ? `transform ${ms('' + transitionDuration)}ms ${easing}` : 'none'
       };
     }
+
     if (!draggable) {
       trackStyle.touchAction = 'auto';
     }
+
     const controls = this.getControls();
+    let arrowOffset = 0;
+    if (this._viewport && this._track) {
+      arrowOffset += this._viewport.offsetHeight / 2;
+      arrowOffset -= this.props.verticalArrowPadding;
+    }
+
+    const customArrowStyle = { ...(isVertical && !arrows) && { transform: `translateY(${ arrowOffset }px)` } };
```

Comment:
`customArrowStyle` is only used for the customArrows component passed by the consumer.

Reply:
removed.